packages/featureflags/FeatureFlagsContext.tsx

```diff
@@ -10,6 +10,7 @@ const FEATURES_RE_FETCH_INTERVAL = 10 * 1000
 
 interface IFeatureFlagsContext {
     useFlag: (name: string) => boolean,
+    useFlagValue: (name: string) => any,
```

Comment:
Maybe provide all available types from growthbook docs? I think there are `number`, `string` and `boolean` and json (for typescript it could be `Record<string, unknown>`

Reply:
```suggestion
    useFlagValue: (name: string) => number | string | boolean | Record<string, unknown>,
```

Reply:
I specified the same type as in the `growthbook` library
```suggestion
    useFlagValue: <T extends JSONValue = any>(name: string) => T | null,
```